packaging/greenboot/functions.sh

```diff
@@ -145,7 +156,7 @@ function namespace_pods_ready() {
 
     # Terminate the script in case more pods are ready than expected - nothing to wait for
     if [ "${tcount}" -gt "${ct}" ] ; then
-        echo "The number of ready pods in the '${ns}' namespace is greater than the expected '${ct}' count. Terminating..."
+        echo "The number of ready pods in the '${ns}' namespace is greater than the expected '${ct}' count. Terminating..." | tee >(systemd-cat -t "${SCRIPT_NAME}")
```

Comment:
Can you add a comment explaining why we need this here for this echo, but not for any of the other echo calls?

Reply:
I included a comment in the same file, above, as its something general: https://github.com/openshift/microshift/pull/2430/files#diff-63bfc037903c0ae357d2459b936ee63776f356d70dd3c4921e2d8e0608009770R13-R21

Reply:
Is it possible to do this at the point where this function is called, instead of inside the function?